docs/tutorial.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: page
+title: Tutorial
+permalink: /tutorial/
+---
+
+In this tutorial, we'll build an example data lake and data pipeline based on airline traffic data published [here](http://stat-computing.org/dataexpo/2009/the-data.html). You can already download and unpack those files. One year is more than enough. You can also quickly grab the data from our [repo](https://github.com/datamindedbe/lighthouse/tree/master/lighthouse-demo/src/main/resources/data).
+
+Steps:
+* Do not remove this line (it will not be displayed)
+{:toc}
+ 
+### Install Lighthouse
+Start by adding the Lighthouse dependency to your project. In case you are using maven:
+```xml
+<dependency>
+    <groupId>be.dataminded</groupId>
+    <artifactId>lighthouse_2.11</artifactId>
+    <version>0.2.0</version>
+</dependency>
+```
+or sbt:
+
+```scala
+libraryDependencies += "be.dataminded" % "lighthouse_2.11" % "0.2.0"
+```
+You can also manually clone and build Lighthouse. You can get the repo here:
+```
+git clone git@github.com:datamindedbe/lighthouse.git
+```
+
+### Define the data in your data lake
+A data lake is a collection of different data sources that we bring together in one place. Building a data lake in Lighthouse means writing Scala code. This has several advantages over more tradtional, statically defined, config files. For one, it allows for more flexibility, which you will see below. Secondly, with your data lake being code, it can go through the same code review and test process as all the rest of your code.  
+
+You need three things to define a data source:
+ 1. **DataLink**: This is a pointer to where the data is stored. More often than not, it's a link to the file system on which you store your data lake (local file system, AWS S3, Azure Blob Storage, Google Cloud Storage, ...). But it could also be a `JDBCDataLink`, connecting to a SQL database. We'll discuss later all the options available.
+ 2. **DataUID**: This uniquely identifies a data source in the data lake. A `DataUID` has 2 parameters: a `namespace` and a `key`. You can think of the namespace as the zone of your data lake where this data belongs, and the key as the actual name of the `DataLink`. How you structure your data lake in zones, is out-of-scope for this tutorial. 
+ 3. **Environment**: Your local development environment will (hopefully) look different from your production environment. In Lighthouse, you can define several environments and add a `DataLink` to one or more environments, depending on your needs. Since all this configuration is code, you can be very flexible in bringing together `DataLinks`, `DataUIDs` and `environments`. 
+ 
+Let's begin with defining some of those data sources. The biggest one is the airplane data itself. For this simple tutorial we define a `FileSystemDataLink` to the folder where we stored the file locally, as such:
```

Comment:
Maybe explain the airplane use case first, before immediately jumping in? As a casual reader I have no idea why you are mentioning airplanes here ;)